src/templates/world_file.cpp

```diff
@@ -73,6 +80,31 @@ bool WorldFile::load(const QString& path)
 	return true;
 }
 
+bool WorldFile::save(const QString &path)
+{
+    if (!loaded) { return false;}
```

Comment:
Please add an empty line after an early `return` so that it becomes more visible.